parsing/builtin_attributes.mli

```diff
@@ -148,9 +155,13 @@ val has_attribute : string list -> Parsetree.attributes -> bool
     "unrolled" only in the case where flambda or flambda2 is configured).  We
     handle this by taking a bool, rather than simply passing fewer nms in those
     cases, to support misplaced attribute warnings - the attribute should not
-    count as misplaced if the compiler could use it in some configuration. *)
+    count as misplaced if the compiler could use it in some configuration.
+
+    [Attributes_filter.t] is an abstract type so that [Builtin_attributes]
+    can expose other filters without calling code inspecting attribute names.
```

Comment:
```suggestion
```

(As discussed, I think this comment is more confusing is helpful - this is an abstract type for the usual reasons.)